lib/restbase.js

```diff
@@ -243,6 +243,10 @@ RESTBase.prototype._request = function(req) {
     var match = priv.router.route(childReq.uri);
     var methods = match && match.value && match.value.methods;
     var handler = methods && (methods[childReq.method] || methods.all);
+    if (childReq.method === 'head' && !handler) {
```

Comment:
@d00rman Hm. Nice idea, but this would imply a little perf overhead with checking `originalMethod` on every request. Do you really see any viable use-cases where separate `HEAD` handler would be needed?